tarballpatches/config.mak.uname.patch

```diff
@@ -1,28 +1,17 @@
 diff --git a/config.mak.uname b/config.mak.uname
-index d63629f..68a7389 100644
+index 64c44db..1caa1f5 100644
 --- a/config.mak.uname
 +++ b/config.mak.uname
-@@ -362,7 +362,7 @@ ifeq ($(uname_S),IRIX)
- 	NO_MMAP = YesPlease
- 	NO_REGEX = YesPlease
- 	SNPRINTF_RETURNS_BOGUS = YesPlease
--	SHELL_PATH = /usr/gnu/bin/bash
-+	SHELL_PATH = "/bin/env bash"
- 	NEEDS_LIBGEN = YesPlease
- endif
- ifeq ($(uname_S),IRIX64)
-@@ -622,6 +622,28 @@ ifeq ($(uname_S),NONSTOP_KERNEL)
+@@ -623,6 +623,24 @@ ifeq ($(uname_S),NONSTOP_KERNEL)
  	SANE_TOOL_PATH = /usr/coreutils/bin:/usr/local/bin
  	SHELL_PATH = /usr/coreutils/bin/bash
  endif
-+
 +ifeq ($(uname_S),OS/390)
-+  V = 2
-+	PERL_PATH = perl
-+	PERL_PATH_FOR_SCRIPTS = /bin/env perl
-+	SHELL_PATH = bash
-+	SHELL_PATH_FOR_SCRIPTS = /bin/env bash
-+	PYTHON_PATH = python
```

Comment:
is python a dependency for git test ? I don't see it in the buildenv

Reply:
Thanks, opened issue https://github.com/ZOSOpenTools/gitport/issues/68.